scripts/channel_map.py

```diff
@@ -6,6 +6,14 @@ class ChannelMap():
             'tfm': 'netcoreapp5.0',
             'branch': 'master'
         },
+        'release/5.0.1xx':{
+            'tfm': 'netcoreapp5.0',
+            'branch': 'release/5.0.1xx'
+        },
+        '5.0':{
+            'tfm': 'netcoreapp5.0',
+            'branch': 'release/5.0'
+        },
         'release/3.1.3xx':{
```

Comment:
not related to the goal of this PR, but should we add `release/3.1.4xx` to the list as well? (we are currently using `release/3.1.3xx`)

Reply:
Thanks for pointing it out! Yes I think we should replace ```release/3.1.3xx``` with ```release/3.1.4xx``` now as there's no active development on the former.. will include the change in this PR.

Reply:
@ooooolivia FWIW if `release/3.1.4xx` is problematic we can just keep using the `release/3.1.3xx` channel. To me, the biggest value of this PR is that it allows me to benchmark 5.0 bits produced from the RC branch, not the 6.0 stuff from master branch.

Reply:
agreed. let's do as much as we can and sort the 3.1.4 thing out later.